contracts/neutron_interchain_queries/src/contract.rs

```diff
@@ -183,6 +200,28 @@ pub fn register_transfers_query(
     Ok(Response::new().add_message(msg))
 }
 
+pub fn register_cw20_balance_query(
+    connection_id: String,
+    update_period: u64,
+    cw20_contract_address: String,
+    account_address: String,
+) -> NeutronResult<Response<NeutronMsg>> {
+    // cw_storage_plus uses this prefix for maps
```

Comment:
There must be an instruction how did you get this prefix, it's not obvious

Reply:
~~Bold of you to assume I remember how I did that.~~

Okay, I will write some extensive docs on how to examine contract storage paths.